### DISCUSSION TITLE
Can't get all devices info through the API

### DISCUSSION THREAD

**Discussion by wypxl** [netbox-community/netbox#14006] [General]
Hi，guys
My NetBox version is 3.6.0, my python version is 3.9.5.

I have about 500 devices registered on netbox. When I got devices info through the API interface 'http://x.x.x.x/api/dcim/devices/' , I could only get information on 60 devices. How can I get info about all my devices?

**Comment by chris240189** [comment]
I was able to get all, when adding "&limit=0" to the API endpoint.

**Reply by wypxl** [reply]
Like 'http://x.x.x.x/api/dcim/devices/&limit=0/'?

**Reply by chris240189** [reply]
https://x.x.x.x/api/dcim/devices/?q=&limit=0

**Reply by candlerb** [reply]
Or https://x.x.x.x/api/dcim/devices/?limit=0
